src/backbone.intercept.js

```diff
@@ -60,9 +60,9 @@ Backbone.Intercept = {
       return;
     }
 
-    // If the Ctrl key is held, we want to open in a new tab.
-    bypass = e.ctrlKey;
-    if (bypass) {
+    // If the platform’s modifier key is held, we want to open in a new tab.
+    var platform = navigator.platform.toLowerCase();
```

Comment:
I wanted to avoid having the Win key (the event’s `metaKey` on Windows) open links in new tabs, which is why I made this code a tad more complicated than what was also my first idea.